### ISSUE TITLE
Fix `NaN` in Final Prediction

### ISSUE THREAD

**Issue by lhl2617** [SOMAS2020/SOMAS2020#334]
`go run .`
```
2021/01/07 20:29:43 [Team3]: Final Prediction: [{NaN NaN NaN 9223372036854775808 NaN}]
2021/01/07 20:29:43 [Team4]: Final Prediction: [{1.747550432276657 1.747550432276657 1.0270893371757925 1 86.75}]
2021/01/07 20:29:43 [Team5]: Final Prediction: [{1.747550432276657 1.747550432276657 1.0270893371757925 1 86.75}]
2021/01/07 20:29:43 [Team6]: Final Prediction: [{NaN NaN NaN 9223372036854775808 NaN}]
2021/01/07 20:29:43 [Team1]: Final Prediction: [{1.747550432276657 1.747550432276657 1.0270893371757925 1 86.75}]
```

Team 6 @Diregie-J  @Regnigh 
Team 3 @preetl @Neelesh99

**Comment by Neelesh99** [comment]
We've got a fix for this in the works, will PR when ready.

**Comment by dejay-vu** [comment]
same here will pr soon

**Comment by lhl2617** [comment]
@Regnigh thanks! resolved from your side in #339

**Comment by lhl2617** [comment]
Team3 solved in #358
